bin/index.js

```diff
@@ -103,6 +104,20 @@ const addFromPackage = (repo, token, path) => {
     .catch(console.warn);
 };
 
+const setGlobalPackage = (path, token) => {
+  fs.writeFile('.git-label-maker-config', pathTool.resolve(__dirname, '..', path), (err) => {
```

Comment:
@EmmaRamirez "labelmaker" should match the name of the repo, like `.git-labelmaker-config' ` 👍